build.savant

```diff
@@ -14,7 +14,7 @@
  * language governing permissions and limitations under the License.
  */
 
-project(group: "com.inversoft.cleanspeak", name: "cleanspeak-containers", version: "4.0.0", licenses: ["ApacheV2_0"]) {
+project(group: "com.inversoft.cleanspeak", name: "cleanspeak-containers", version: "4.0.1", licenses: ["ApacheV2_0"]) {
```

Comment:
This version should really be the same as `cleanspeak-app`. And if we haven't released `4.0.0` - I would assume this would then still be `4.0.0`. Is that correct, or is this something else?

Reply:
This was accidental. I was playing with `is version`. Fixed.